packages/vite/package.json

```diff
@@ -19,7 +19,10 @@
   "scripts": {
     "build": "yarn build:js && yarn build:types",
     "build:js": "babel src -d dist --extensions \".js,.ts,.tsx\"",
-    "build:types": "tsc --build --verbose"
+    "build:types": "tsc --build --verbose",
+    "test": "yarn test:node && echo",
```

Comment:
CI will pass `--colors --maxWorkers` to `yarn test`. The node testrunner doesn't have those flags so it will error out when running. Adding `&& echo` "protects" the testrunner from those flags.